test/test.js

```diff
@@ -9,6 +9,12 @@ describe('Tex in-line math', function () {
   it('should work properly', function () {
     md.render('$1 *2* 3$').should.eql('<p>\\(1 *2* 3\\)</p>\n')
   })
+  it('can handle dollors in formulas', function () {
+    md.render('$sum_{$4}^2$').should.eql('<p>\\(sum_{$4}^2\\)</p>\n')
+  })
+  it('can handle multiple dollors in formulas', function () {
+    md.render('if $a^{$2.00}$ then $b^{$3.00}$').should.eql('<p>if \\(a^{$2.00}\\) then \\(b^{$3.00}\\)</p>\n')
+  })
```

Comment:
always nice to see !

Reply:
@tatums you spelled "dollars" incorrectly.

Reply:
misspelled "dollars"

Reply:
Oooppps!